modules/ROOT/pages/index.adoc

```diff
@@ -107,7 +107,7 @@ Defines the structure of the transactions that the host sends to the backend thr
 +
 Source at Host 
 +
-Defines the structure of the transactions that the backend sends to the host through outbound message flows. You can define custom message attributes for this message type.
+Defines the structure of the transactions that the backend sends to the host through outbound message flows. You can define custom message attributes for this message type. You can define <<custom-attributes,custom message attributes>> for this message type.
```

Comment:
duplicated sentence.

Reply:
These are two different things - endpoints and message types. I will qualify these so that it won't be confusing.